gui_qt/toast_widget.py

```
@@ -0,0 +1,248 @@
+        self,
+        parent: QWidget,
+        message: str,
+        style: ToastStyle = ToastStyle.SUCCESS,
+        duration_ms: int = 2500,
+    ) -> None:
+        super().__init__(parent)
+        self._style = style
+        self._duration_ms = duration_ms
+        self._is_dark = self._detect_dark_mode()
+
+        # 窗口标志：无边框提示
```

Ruff (0.15.20)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)